### PR TITLE
[MRG] propagate `verbose` in `write_raw_bids` to `utils._read_events`

### LINKED ISSUE #448: verbose=False is not propagated to the function that finds events
#### Describe the bug
Calling `write_raw_bids` with `verbose=False` still produces output related to finding events.

#### Steps to reproduce
```python

from tempfile import mkdtemp
from pathlib import Path

import mne
import mne_bids

test_elekta = Path(mne.datasets.testing.data_path(verbose=True)) / 'misc' / 'test_elekta_3ch_raw.fif'
raw = mne.io.read_raw_fif(test_elekta, verbose=False)
bids_root = mkdtemp()
mne_bids.write_raw_bids(raw, 'sub-01_ses-01_run-05', bids_root=bids_root, verbose=False)
```

#### Expected results
Nothing is printed to the console.

#### Actual results
The following is printed to the console:
```
162 events found
Event IDs: [  1   2   4   8  16  32  64 128]
```

#### Additional information
`mne.sys_info()`:
```
Platform:      Windows-10-10.0.18362-SP0
Python:        3.7.4 (default, Aug  9 2019, 18:34:13) [MSC v.1915 64 bit (AMD64)]
Executable:    D:\software_windows\miniconda3\envs\syntax_tagging\python.exe
CPU:           Intel64 Family 6 Model 69 Stepping 1, GenuineIntel: 4 cores
Memory:        Unavailable (requires "psutil" package)
mne:           0.19.2
numpy:         1.16.3 {blas=mkl_rt, lapack=mkl_rt}
scipy:         1.2.1
matplotlib:    3.2.1 {backend=Qt5Agg}

sklearn:       0.21.3
numba:         Not found
nibabel:       3.1.0
cupy:          Not found
pandas:        0.25.1
dipy:          1.1.0
mayavi:        4.7.1 {qt_api=pyqt5, PyQt5=5.9.2}
pyvista:       Not found
vtk:           8.2.0
```
`mne_bids.__version__`:
```
'0.4'
```
But the result is the same with the current development version '0.5.dev0'.

### PR DESCRIPTION
**Pull request by kalenkovich**
PR Description
--------------

The parameter `verbose` of `write_raw_bids` was not passed to `utils._read_raw_bids` which didn't have the `verbose` parameter at all. Consequently, `utils._read_raw_bids` did not pass the non-existent parameter to `read_events`, `find_events`, and `events_from_annotations`.

What has been done:

- added `verbose` parameter to `utils._read_raw_bids`,
- passed `verbose` in `write_raw_bids` to `utils._read_raw_bids`
- passed `verbose` in `utils._read_raw_bids` to `read_events`, `find_events`, and `events_from_annotations`

closes #448 

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [x] All comments resolved
- [x] This is not your own PR
- [x] All CIs are happy
- [x] PR title starts with [MRG]
- [x] [whats_new.rst](https://github.com/mne-tools/mne-bids/blob/master/doc/whats_new.rst) is updated
- [x] PR description includes phrase "closes <#issue-number>"
- [x] Commit history does not contain any merge commits

### CODE REVIEW

File: mne_bids/utils.py [hunk 1]
```diff
@@ -726,6 +726,7 @@ def _read_events(events_data, event_id, raw, ext):
         The data as MNE-Python Raw object.
     ext : str
         The extension of the original data file.
+    %(verbose)s
```
Comment (agramfort) [inline mne_bids/utils.py:729]: this will not work as you don't have a fill_doc decorator on the function _read_events

File: mne_bids/utils.py [hunk 2]
```diff
@@ -747,9 +748,9 @@ def _read_events(events_data, event_id, raw, ext):
                              'found %s' % events_data.shape[1])
         events = events_data
     elif 'stim' in raw:
-        events = find_events(raw, min_duration=0.001, initial_event=True)
+        events = find_events(raw, min_duration=0.001, initial_event=True, verbose=verbose)
```
Comment (agramfort) [inline mne_bids/utils.py:751]: check flake8 errors. I suspect this line is too long

File: mne_bids/utils.py (hunk 1 above)
Comment (kalenkovich) [reply mne_bids/utils.py:730]: Added `@verbose` decorator. Not sure if that is the right way: just copied the decorator from `read_events`, `find_events`, and `events_from_annotations`.

Comment (jasmainak) [reply mne_bids/utils.py:730]: Just document it manually. You might need the function `fill_docs` to make it work.

File: mne_bids/utils.py (hunk 2 above)
Comment (kalenkovich) [reply mne_bids/utils.py]: You are absolutely correct, This and two more lines were too long - fixed.

**Comment by jasmainak** [comment]
@kalenkovich can you update `whats_new.rst` when you're done?

and please refer to the checklist. You need to link to the issue that you are closing

**Comment by kalenkovich** [comment]
> @kalenkovich can you update `whats_new.rst` when you're done?

I can't find this file in `mne_bids`.

> and please refer to the checklist. You need to link to the issue that you are closing

I didn't think to look at it because it was addressed to the maintainer.

I changed the title to start with "[MRG]" and added "closes #448" to the description.

**Comment by jasmainak** [comment]
> I can't find this file in mne_bids.

here you go: https://github.com/mne-tools/mne-bids/blob/master/doc/whats_new.rst. It's also linked in the checklist above!

> I didn't think to look at it because it was addressed to the maintainer.

oops :) how can we improve the wording?

**Comment by kalenkovich** [comment]
Apologies, @jasmainak - found "whats_new.rst". Github search betrayed me.

**Comment by kalenkovich** [comment]
> oops :) how can we improve the wording?

I would add a similar checklist to "contributing.md" and refer to it in the pull request template.

**Comment by jasmainak** [comment]
humm but that would defeat the purpose of a checklist. I use it every time to make sure everything is in order before merging. I do expect the contributor to look at it too. I guess we can leave it like this for now. We'll change it if there are more complaints :)

**Comment by agramfort** [comment]
@kalenkovich see that CIs are not happy.

you can see the circleci docs-build artifact details to check if you're what's new entry looks correct?

**Comment by kalenkovich** [comment]
> @kalenkovich see that CIs are not happy.
> 
> you can see the circleci docs-build artifact details to check if you're what's new entry looks correct?

I did not add myself to the "authors" list - fixed now.

File: mne_bids/utils.py (hunk 1 above)
Comment (kalenkovich) [reply mne_bids/utils.py]: Documented manually. Copied the text from `mne.utils.docs.docdict['verbose']`.

File: doc/whats_new.rst [hunk 3]
```diff
@@ -30,6 +30,7 @@ Changelog
 - :func:`mne_bids.write_raw_bids` now writes CoordinateSystemDescription as specified in BIDS Specification if CoordinateSystem is MNE-compatible, by `Adam Li`_ (`#416 <https://github.com/mne-tools/mne-bids/pull/416>`_)
 - :func:`mne_bids.write_raw_bids` and :func:`mne_bids.read_raw_bids` now handle scalp EEG if Captrak coordinate system and NAS/LPA/RPA landmarks are present, by `Adam Li`_ (`#416 <https://github.com/mne-tools/mne-bids/pull/416>`_)
 - :func:`mne_bids.get_matched_empty_room` now implements an algorithm for discovering empty-room recordings that do not have the recording date set as their session, by `Richard Höchenberger`_ (`#421 <https://github.com/mne-tools/mne-bids/pull/421>`_)
+- :func:`mne_bids.write_raw_bids` now applies `verbose` to the functions that read events, by `Evgenii Kalenkovich`_ (`#453 <https://github.com/mne-tools/mne-bids/pull/453>`_)
```
Comment (jasmainak) [inline doc/whats_new.rst:33]: I believe this should go in the Bug section of whats_new

Comment (kalenkovich) [reply doc/whats_new.rst]: Moved it to Bug section.

File: mne_bids/utils.py [hunk 4]
```diff
@@ -726,6 +726,9 @@ def _read_events(events_data, event_id, raw, ext):
         The data as MNE-Python Raw object.
     ext : str
         The extension of the original data file.
+    verbose : bool | str | int | None
+        If not None, override default verbose level (see :func:`mne.verbose`
+        and :ref:`Logging documentation <tut_logging>` for more).
```
Comment (sappelhoff) [inline mne_bids/utils.py:731]:
```suggestion
        If not None, override default verbose level (see :func:`mne.verbose`).
```

the sphinx ref to the mne docs won't work in mne-bids, so I suggest to remove it.

Comment (agramfort) [reply mne_bids/utils.py:731]: +1

**Comment by agramfort** [comment]
CIs complain https://github.com/mne-tools/mne-bids/pull/453/checks?check_run_id=793001258

**Comment by agramfort** [comment]
@sappelhoff merge if ok for you too. thx @kalenkovich

**Pull request merged**